tests/admin_inlines/models.py

```diff
@@ -293,9 +293,18 @@ class SomeChildModel(models.Model):
     parent = models.ForeignKey(SomeParentModel, models.CASCADE)
     readonly_field = models.CharField(max_length=1)
 
-# Other models
 
+# Models for #18549
+class Car(models.Model):
```

Comment:
Are there other models we can use to avoid adding more?

Reply:
There was only one other model with a `OneToOneField` in it. And I needed to register one model with a `StackedInline` and one with a `TabularInline`.

I suppose I could add a `OneToOneField` to an existing model though, and be careful that's it's not going to affect any of the tests that are already using it.

Thanks for the suggestion. I'll see if I can do the above.

Reply:
@smithdc1 I've now removed all models this PR added, and just adapted one existing model in a way that doesn't affect other tests.